2-mandatory/6-writing-tests-advanced.js

```diff
@@ -56,6 +56,16 @@ function formatCourseworkResult(trainee) {
   }
 */
 
+test("Traine Xin, score of 63", () => {
+  const trainee = {
+    name: `Xin`,
+    score: 63,
+  };
+  expect(formatCourseworkResult(trainee)).toEqual(
+    `Xin's coursework was marked as grade C.`
+  );
+});
```

Comment:
looks good! Could you think of a way to consolidate this? 

You could you try something like-- 

```
let result = formatCourseworkResult({
name: 'Xin',
score: 63})
```

Could you finish the `expect` function?